clouddriver/clouddriver-artifacts/clouddriver-artifacts-docker/src/main/java/com/netflix/spinnaker/clouddriver/artifacts/docker/HelmOciArtifactConfiguration.java

```diff
@@ -36,10 +36,15 @@
 @Slf4j
 class HelmOciArtifactConfiguration {
 
+  @Bean
```

Comment:
This doesn't seem like a refactor.  Is it a fix or a feature?  If it's a fix, can we have a test that fails first?  I guess MySqlConfigurationTest sort of counts, but it would be nice to introduce DockerArtifactConfigurationTest in a separate commit that demonstrates the bug.

I'll give it a shot.